### DISCUSSION TITLE
I have an error with JWT auth

### DISCUSSION THREAD

**Discussion by vlad99902** [loopbackio/loopback-next#7503] [Q&A]
Hello everyone. I need to add JWT auth to my loopback api. 
I just tried to use this tutorual: https://loopback.io/doc/en/lb4/Authentication-tutorial.html#step-1-bind-jwt-component-in-the-application

In my new lb4 api it works fine, but in oldest api i have an error, when i tried to /signup:
```Request POST /signup failed with status code 500. TypeError: this.modelClass.create is not a function```

package.json
```
"dependencies": {
    "@loopback/authentication": "^7.3.0",
    "@loopback/authentication-jwt": "^0.9.0",
    "@loopback/boot": "^3.1.2",
    "@loopback/core": "^2.13.1",
    "@loopback/repository": "^3.3.0",
    "@loopback/rest": "^9.1.1",
    "@loopback/rest-explorer": "^3.0.5",
    "@loopback/service-proxy": "^3.0.5",
    "loopback-connector-kv-redis": "^3.0.3",
    "loopback-connector-rest": "^3.7.0",
    "tslib": "^2.0.0"
  },
  "devDependencies": {
    "@loopback/build": "^6.2.8",
    "@loopback/eslint-config": "^10.0.4",
    "@loopback/testlab": "^3.2.10",
    "@types/node": "^10.17.48",
    "eslint": "^7.15.0",
    "source-map-support": "^0.5.19",
    "tsc-watch": "^4.2.9",
    "typescript": "~4.1.2"
  }
```

**Comment by vlad99902** [comment]
UPD.
I found that if I'm using JSON file as datasourse everything works fine.
But if I'm trying to use Redis as datasourse I have an error, that I wrote before.

Hope that someone will help me :)

**Comment by ewertonrfd** [comment]
Hey,
I don't know if it will be exactly your problem
But when adding JWT I also had a problem

Then I created a mini-tutorial to always fix the bug (there is already an issue open)
https://github.com/WebEvolui/JWT-in-Loopback4 (pt-br)

If you followed the complete tutorial, only the above already solves
